client/components/data/query-theme-filters/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestThemeFilters } from 'state/themes/actions';
+
+export class QueryThemeFilters extends Component {
+	static propTypes = {
+		requestThemeFilters: PropTypes.func.isRequired,
+	}
+
+	componentWillMount() {
+		this.props.requestThemeFilters();
+	}
```

Comment:
Hah, funny to see a query component with no `componentWillReceiveProps` -- since this one just doesn't take any props...